whois-api/src/main/java/net/ripe/db/whois/api/mail/dequeue/MessageService.java

```diff
@@ -100,7 +100,8 @@ public void verifyAndSetAsUnsubscribed(final EmailMessageInfo message) {
 
     private boolean isValidMessage(final EmailMessageInfo message, final List<String> outgoingEmail) {
         if (message.messageId() == null || message.emailAddresses() == null || message.emailAddresses().isEmpty()) {
-            LOGGER.warn("Incorrect message {}", message.messageId());
+            LOGGER.info("Incorrect message {}", message.messageId());
+            LOGGER.warn("Incorrect message <Filtered>");
```

Comment:
Can you keep the message-id ? I don't believe it's PII as it's auto-generated to identify an email message. It makes debugging harder if we can't identify the message